### PR TITLE
Handle logic info page

### PR DESCRIPTION
**Pull request by 31191026446**

### CODE REVIEW

File: VMS/Pages/ActivityInfoPage/InfoPage.razor [hunk 1]
```diff
@@ -1,70 +1,43 @@
...
+﻿@if (Activity is { } activity)
+{
+    <div class="infopage">
+        <div class="infopage-title">
+            <p class="title__content">
+                @activity.Name
+            </p>
+            <div class="title__name">
+                <span class="material-icons"><span class="earth-icon">public</span></span>
+                <p class="organization m-0">@activity.Organizer?.UserName</p>
```
Comment (hoaitran308) [inline VMS/Pages/ActivityInfoPage/InfoPage.razor:10]: this should be name of Org, not user name (email)

**Comment by LongDuongBIT** [comment]
Please continue review this PR.

File: VMS.Application/Services/ActivityService.cs [hunk 2]
```diff
@@ -311,6 +319,28 @@ public async Task<List<UserWithActivityViewModel>> GetRelatedActivities(string u
             return _mapper.Map<List<UserWithActivityViewModel>>(activities);
         }
 
+        public async Task<List<ViewActivityViewModel>> GetOtherActivities(string orgId, int[] excludedActitivyIds)
```
Comment (hoaitran308) [inline VMS.Application/Services/ActivityService.cs:322]: Please end with "Async" prefix.

File: VMS.Application/Services/ReportService.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{
+    public class ReportService : BaseService, IReportService
+    {
+        public ReportService(IIdentityService identityService,
+                                   IRepository repository,
+                                   IDbContextFactory<VmsDbContext> dbContextFactory,
+                                   IMapper mapper) : base(repository, dbContextFactory, mapper)
+        {
+        }
+        public async Task AddReport(ReportViewModel reportViewModel)
```
Comment (hoaitran308) [inline VMS.Application/Services/ReportService.cs:21]: Please end with "Async" prefix.

File: VMS.Application/Services/ReportService.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+using VMS.Application.ViewModels;
+using VMS.Domain.Interfaces;
+using VMS.Domain.Models;
+using VMS.Infrastructure.Data.Context;
+
+namespace VMS.Application.Services
+{
+    public class ReportService : BaseService, IReportService
+    {
+        public ReportService(IIdentityService identityService,
```
Comment (hoaitran308) [inline VMS.Application/Services/ReportService.cs:15]: identityService never used. Should we remove it?

File: VMS.Application/Services/ActivityService.cs (hunk 2 above)
Comment (LongDuongBIT) [reply VMS.Application/Services/ActivityService.cs]: Fixed!

File: VMS.Application/Services/ReportService.cs (hunk 3 above)
Comment (LongDuongBIT) [reply VMS.Application/Services/ReportService.cs]: Fixed!

File: VMS.Application/Services/ReportService.cs (hunk 4 above)
Comment (LongDuongBIT) [reply VMS.Application/Services/ReportService.cs]: Make sense, fixed!

**Review by hoaitran308 — APPROVED** [review]
Now you have the hang of it.

**Pull request merged**